electron_app/helpers/_handleDisk.js

```diff
@@ -0,0 +1,27 @@
+const fs = require('fs');
+const mkdirp = require('mkdirp');
+
+const handleFolder = (directory, item) => {
+    let data = {
+            'path': directory,
+            'item': item
+        };
+    return new Promise((resolve, reject) =>{
+        if (!fs.existsSync(directory)) {
+            mkdirp.sync(directory, (err) => {
+                if (err) {
+                    console.log(err);
+                    rejec(err);
+                }
+                else {
+                    console.log('Created directory at ', directory);
```

Comment:
I assume this log is for debugging, can it be removed now?

Reply:
Sure.

Reply:
About the same, what do you think of lines 18 and 23 in the DownloadManager?